source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py

```diff
@@ -217,9 +217,19 @@ class StopAreaSerializer(serpy.DictSerializer):
     timezone = NestedPropertyField(attr='properties.geocoding.timezone')
     commercial_modes = NestedDictGenericField(attr='properties.geocoding.commercial_modes', many=True)
     physical_modes = NestedDictGenericField(attr='properties.geocoding.physical_modes', many=True)
+    comments = NestedDictCommentField(attr='properties.geocoding.comments', many=True)
+    comment = jsonschema.MethodField(display_none=True)
```

Comment:
in jormungandr: `comment = FirstCommentField(attr='comments', display_none=False, deprecated=True)`

I need a corresponding test.

Reply:
l have tried to use this method but it didn't work.
This is the reason why I have created the "get_comment" method in StopAreaSerializer class.

Reply:
I have tried to use this method, but it didn't work. This is the reason why I have created "get_comment" method in StopAreaSerializer class.
There is a test : https://github.com/CanalTP/navitia/pull/2378/files#diff-e965ac31a14eb9d613cd1e9053897620R1179

Reply:
JIRA avec les explications associés :
https://jira.canaltp.fr/browse/NAVP-810

Reply:
No, I want to say that, if there is no comment, the comment field must not be present.

Reply:
`display_none=False`

Reply:
With this PR, when stop area comments exist we display it, but if there is no comments associated, "comments" and "comment" fiied no exist.

A example with 2 stops area, the first with comments and the second without :

![with_and_without_comment](https://user-images.githubusercontent.com/32099204/38089486-d92447ac-335f-11e8-9081-fdf7eb8cbd67.png)

Reply:
I can add a test when a stop area has no comments.

Reply:
Added test :white_check_mark:

Reply:
:+1:

Reply:
Did you try without the `display_none` ? It's probably useless, if it works the way we want, so we might as well remove it.